inspirehep/dojson/common/base.py

```diff
@@ -21,11 +21,12 @@
 # or submit itself to any jurisdiction.
 
 """MARC 21 model definition."""
+import six
```

Comment:
And this is what happens when I browse the entire PR instead of going commit by commit. Scratch that.